app/views/staff/shared/activities/tree_view/_table_tabbed.html.haml

```diff
@@ -32,4 +32,4 @@
                       - third_party_projects.each do |third_party_project|
                         = render partial: "staff/shared/activities/tree_view/row", locals: { activity: third_party_project, parent: project, is_parent: false }
 
-              = link_to "View all activities", activities_path(organisation_id: current_user.organisation.id), class: "govuk-body govuk-link"
+              = link_to "View all activities", activities_path(organisation_id: current_user.organisation.id), role: "tab", class: "govuk-body govuk-link"
```

Comment:
I don't think this is right. The link to View all activities opens a new page, and is not actually a tab. (See annotated screenshot above.)